src/main/java/application/rest/v1/configmaps/ConfigMapCache.java

```diff
@@ -191,8 +193,30 @@ public static V1ConfigMap getConfigMap(ApiClient client, String namespace, Strin
         try {
             CoreV1Api api = new CoreV1Api();
             api.setApiClient(client);
-
             V1ConfigMap map = api.readNamespacedConfigMap(name, namespace, null, null, null);
+
+            // issue warning if kappnav.io/map-type label is not set
+            V1ObjectMeta metadata = map.getMetadata();
```

Comment:
You may want to wrap this entire section in "if (Logger.isWarningEnabled())" since this code is only being used to compute a warning message.

Reply:
code updated.